example/prometheus.yaml

```diff
@@ -0,0 +1,10 @@
+global:
+  scrape_interval: 15s
+scrape_configs:
+  - job_name: "envoy-stats"
+    metrics_path: /stats/prometheus
+    scrape_interval: 5s
+    static_configs:
+      - targets: ["envoy:8082"]
```

Comment:
I think I would go ahead and put all the files under `envoy` or a subdirectory of it. Many parts of the example are tied to it being used in that docker-compose file so treating them as independent examples looks a bit confusing.